interfaces/AdminDaoI.ts

```diff
@@ -2,6 +2,7 @@ import User from "../models/users/User";
 
 export default interface AdminDaoI {
     findAllUsers(): Promise<User[]>;
+    blockUser(uid: string, user: User): Promise<User[]>;
```

Comment:
I guess `blockUser` should return a single user?

Reply:
Thanks for pointing it out. Will change it in the next PR.